MAINTAINERS.md

```diff
@@ -103,4 +103,10 @@ Typically, this will be an approval from another maintainer, often the Docs Lead
 
 Lastly, we have a "DocsBot" that posts congratulation messages in the #docs Discord channel. **Our bot also acknowledges co-authors!** During "Squash and Merge," if you would like to acknowledge any reviewer, please type in the exact phrase `Co-authored-by:` and include any missing reviewers or contributors whom you'd like to acknowledge!
 
-TODO: List the steps to get the proper name to use in commit message.
+### Getting `Co-Authored-by` Commit Message Name and Email
+
+In order to ensure that we're using the proper commit co-author's name and email associated with their GitHub account, we can look up that information via a GitHub commit.
```

Comment:
```suggestion
For the bot to work properly, you must use the exact co-author's name and email associated with their GitHub account. 

You can look up that information using a GitHub commit they have made from any PR, even one from another repository! Please don't hesitate to take the extra step and include others!
```

Thanks @crutchcorn ! I took the liberty of making this a bit more "threatening" (i.e. "you must follow protocol or it won't work!") and changed the verb tense to match the rest, which is telling "you" how to do things.